web/massastation/src/components/SidePanels/Upload/Upload.tsx

```diff
@@ -49,13 +54,27 @@ export default function Upload() {
     mutate: mutableUpload,
     isLoading: uploadLoading,
     error: uploadFail,
-  } = usePut<FormData, IUploadResponse>('websiteUploader/prepare', {
-    'Content-Type': 'multipart/form-data',
-  });
+  } = usePut<FormData, IUploadResponse, AxiosError<IUploadError>>(
+    'websiteUploader/prepare',
+    {
+      'Content-Type': 'multipart/form-data',
```

Comment:
Niiiice it accepts multipart/form-data too! We can make it easy after creating a auto detect under the usePut. Cool!